modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -214,7 +214,7 @@ Then the copied tables were deleted after the upgrade was completed.
 This doubled the upgrade time, so admins could skip this test (by
 risking a failed upgrade) with `php occ upgrade --skip-migration-test`.
 
-The migration test has been removed from ownCloud 9.2. ownCloud server
+The migration test has been removed from ownCloud 10.0. ownCloud server
```

Comment:
As the original change was referring specifically to ownCloud 9.2, I wouldn't update the version number. Instead, I'd change the sentence to: "The migration test has been removed."